From 7a3c66b8b4a5a1d939e45981da093080c4dc0c2c Mon Sep 17 00:00:00 2001
From: Kevin W Matthews <kevinwmatthews@gmail.com>
Date: Sun, 30 Aug 2026 12:30:21 -0600
Subject: [PATCH] create changes in Yocto build environment

---
 main.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/main.c b/main.c
index 6afbae4..ecb8524 100644
--- a/main.c
+++ b/main.c
@@ -2,5 +2,6 @@
 
 int main(void) {
     printf("Hello from C!\n");
+    printf("This is modified from the Yocto project.\n");
     return 0;
 }
